packages/react-core/src/demos/Banner.md

```diff
@@ -48,13 +38,13 @@ class BannerDemo extends React.Component {
             </Flex>
           </Banner>
           <PageSection>
-            <TextContent>
-              <Text component="h1">Main title</Text>
-              <Text component="p">
+            <Content>
+              <h1>Main title</h1>
+              <p>
                 Body text should be Overpass Regular at 16px. It should have leading of 24px because <br />
```

Comment:
```suggestion
                Body text should be Red Hat Text at 1rem(16px). It should have leading of 1.5rem(24px) because <br />
```
this aligns with our higher level typography rules/docs